.prettierignore

```diff
@@ -2,4 +2,5 @@ packages/*/dist
 packages/*/api-docs
 packages/cli/generators/*/templates
 *.json
-*.md
+CHANGELOG.md
+.sandbox
```

Comment:
can we add `packages/*/CHANGELOG.md` here?

Reply:
I think `CHANGELOG.md` already covers nesting files with the same name.